src/main/java/com/todolist/app/springmvcboottodolist/service/ToDoListStore.java

```diff
@@ -0,0 +1,79 @@
+package com.todolist.app.springmvcboottodolist.service;
+
+import com.todolist.app.springmvcboottodolist.models.Item;
+import com.todolist.app.springmvcboottodolist.models.Priority;
+import com.todolist.app.springmvcboottodolist.models.Tag;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Service;
+import com.todolist.app.springmvcboottodolist.models.Status;
+import java.util.*;
+
+@Service
+public class ToDoListStore implements Store{
+    private static final Map<String, Item> items = new LinkedHashMap<>();
+
+    static {
```

Comment:
Since we are in the Spring world, let's use @PostConstruct method, e.g. https://www.baeldung.com/spring-postconstruct-predestroy